pyiron_gui/project/project_browser.py

```diff
@@ -97,15 +98,370 @@ def __getattr__(self, item):
+        self._box.children = tuple([self._header, self._output])
+
+    def _init_option_widgets(self):
+
+        self._option_widgets = {
+            'camera': widgets.Dropdown(options=['perspective', 'orthographic'], value=self._options['camera'],
+                                       layout=widgets.Layout(width='min-content'), description_tooltip='Camera mode'),
+            'particle_size': widgets.FloatSlider(value=self._options['particle_size'],
+                                                 min=0.1, max=5.0, step=0.1, readout_format='.1f',
+                                                 description="atom size",
+                                                 layout=widgets.Layout(width='60%')),
+            'cell': widgets.Checkbox(description='cell', indent=False,
+                                     value=self._options['cell'],
+                                     description_tooltip='Show cell if checked'),
+            'axes': widgets.Checkbox(description='axes', indent=False,
+                                     value=self._options['axes'],
+                                     description_tooltip='Show axes if checked'),
+            'reset_view': widgets.Checkbox(description='reset view', indent=False,
+                                           value=self._options['reset_view'],
+                                           description_tooltip='Reset view if checked')
+        }
+
+    @property
+    def _option_representation(self):
+        """ipywidet to change the options for the self_representation"""
+        widget_list = list(self._option_widgets.values())
+        return widgets.VBox([
+            widgets.HBox(widget_list[0:2]),
+            widgets.HBox(widget_list[2:])
+        ])
+
+    def _parse_option_widgets(self):
+        for key in self._options.keys():
+            self._options[key] = self._option_widgets[key].value
+
+    def _update_ngl_widget(self):
+        self._parse_option_widgets()
+        if self._ngl_widget is not None:
+            orient = self._ngl_widget.get_state()['_camera_orientation']
+        else:
+            orient = []
+
+        self._ngl_widget = self._obj.plot3d(
+            mode='NGLview',
+            show_cell=self._options['cell'],
+            show_axes=self._options['axes'],
+            camera=self._options['camera'],
+            spacefill=True,
+            particle_size=self._options['particle_size'],
+            select_atoms=None,
+            background="white",
+            color_scheme=None,
+            colors=None,
+            scalar_field=None,
+            scalar_start=None,
+            scalar_end=None,
+            scalar_cmap=None,
+            vector_field=None,
+            vector_color=None,
+            magnetic_moments=False,
+            view_plane=np.array([0, 0, 1]),
+            distance_from_camera=1.0,
+            opacity=1.0
+        )
+        if not self._options['reset_view'] and len(orient) == 16:
```

Comment:
What's 16 stand for? Some 4x4 matrix?  Can stay, but may be a comment is helpful for future people who touch this.

Reply:
Good point, I actually already have to look it up again... There we go, I had it from our `plot3d` implementation [here](https://github.com/pyiron/pyiron_atomistics/blob/64b7216b3d4d8ab0e535191e70c482a341a1d286/pyiron_atomistics/atomistics/structure/_visualize.py#L690).